website/templates/log_templates.mako

```diff
@@ -95,8 +95,11 @@ edited description of  <a class="log-node-title-link" data-bind="attr: {href: no
 </script>
 
 <script type="text/html" id="updated_fields">
-updated the <span data-bind="listing: params.updated_fields"></span> of 
-<a class="log-node-title-link" data-bind="attr: {href: nodeUrl}, text: nodeTitle"></a>
+  changed the <span data-bind="listing: {
+                                 data: params.updated_fields,
+                                 map: function(key, item) { return key + ' to ' + item.new;}
```

Comment:
This displays the internal representation for categories, not the user-friendly representations.